docs/conf/portland/2020/news/schedule.rst

```diff
@@ -0,0 +1,59 @@
+:template: {{year}}/generic.html
+
+.. post:: July 17, 2020
+   :tags: {{year}}, portland-{{year}}, schedule
+
+Announcing Full Schedule
+========================
+
+Hey everyone!
+
+We're less than a month away from the first ever online Portland conference and we're working hard to make sure it'll be as welcoming and as fun as the in-person event you're probably familar with.
+
+Today we're announcing the full presentation schedule and the onnline platform we'll be hosting the conference on.
+
+Conference platform
+--------------------
+
+The conference will be held on https://hopin.to/, which not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and impromptu 1-on-1 meetings. We're updating our welcome guide so it'll all feel familar before you've even arrived.
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the `talk schedule <https://www.writethedocs.org/conf/portland/{{year}}/schedule/>`_ for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. We're excited to have Q&A after most talks
+
+As always, we'll be running the `unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_, and `lightning talks <https://www.writethedocs.org/conf/portland/2019/lightning-talks/>`_ each day after lunch.
```

Comment:
We should either cut this section since we mention all this in detail below, or add the job fair here.

Reply:
I think it's best to remove both of these links, since we're listing job fair and everything else later in the post.